src/bin/vip-import.js

```
@@ -240,15 +240,24 @@ program
 	.description( 'Import SQL to a VIP Go site' )
 	.option( '-t, --throttle <mb>', 'SQL import transfer limit in MB/s', 1, parseFloat )
 	.option( '-s, --skip-confirm', 'Skip the confirmation step' )
+	.option( '--from <from>', 'Search/Replace' )
+	.option( '--to <to>', 'Search/Replace' )
 	.action( ( site, file, options ) => {
 		try {
 			which.sync( 'mysql' );
 		} catch ( e ) {
 			return console.error( 'MySQL client is required and not installed.' );
 		}
 
+		let searchReplace = {};
+
+		if ( options.from && options.to ) {
```

We should note that the import will run search-replace including the from/to values in the confirm.

---

Should probably also throw an error if only one of the options is provided.